### ISSUE TITLE
Source install fails on Windows

### ISSUE THREAD

**Issue by dhirschfeld** [jupyter/help#47]
Running `python setup.py js css` fails for me with the below error:
```
[py-dev] C:\dev\code\notebook (master) (jupyter-notebook-deps@4.0.0)
λ python setup.py js css

<SNIP>

running css
> npm run build:css

> jupyter-notebook-deps@4.0.0 build:css C:\dev\code\notebook
> concurrent 'npm run build:css:ipython' 'npm run build:css:style'

[0] 'np' is not recognized as an internal or external command,
[0] operable program or batch file.
[1] 'run' is not recognized as an internal or external command,
[1] operable program or batch file.
[2] The filename, directory name, or volume label syntax is incorrect.
[3] 'np' is not recognized as an internal or external command,
[3] operable program or batch file.
[4] 'run' is not recognized as an internal or external command,
[4] operable program or batch file.
[5] The filename, directory name, or volume label syntax is incorrect.
[4] run exited with code 1
[3] np exited with code 1
[2] build:css:ipython' exited with code 1
[1] run exited with code 1
[0] np exited with code 1
[5] build:css:style' exited with code 1

npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "build:css"
npm ERR! node v6.2.0
npm ERR! npm  v3.8.9
npm ERR! code ELIFECYCLE
npm ERR! jupyter-notebook-deps@4.0.0 build:css: `concurrent 'npm run build:css:ipython' 'npm run build:css:style'`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the jupyter-notebook-deps@4.0.0 build:css script 'concurrent 'npm run build:css:ipython' 'npm run build:css:style''.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the jupyter-notebook-deps package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     concurrent 'npm run build:css:ipython' 'npm run build:css:style'
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs jupyter-notebook-deps
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls jupyter-notebook-deps
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\dev\code\notebook\npm-debug.log
Traceback (most recent call last):
  File "setup.py", line 199, in <module>
    main()
  File "setup.py", line 196, in main
    setup(**setup_args)
  File "C:\Python\envs\py-dev\lib\distutils\core.py", line 148, in setup
    dist.run_commands()
  File "C:\Python\envs\py-dev\lib\distutils\dist.py", line 955, in run_commands
    self.run_command(cmd)
  File "C:\Python\envs\py-dev\lib\distutils\dist.py", line 974, in run_command
    cmd_obj.run()
  File "C:\dev\code\notebook\setupbase.py", line 412, in run
    run(['npm', 'run', 'build:css'])
  File "C:\dev\code\notebook\setupbase.py", line 333, in run
    return check_call(cmd, *args, **kwargs)
  File "C:\Python\envs\py-dev\lib\subprocess.py", line 584, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['npm', 'run', 'build:css']' returned non-zero exit status 1
```

Any idea what is wrong? *Should* this work on Windows?

NB: The `npm-debug.log` doesn't exist.

**Comment by Carreau** [comment]
in `package.json`, can you try to replace :
`"build:css": "concurrent 'npm run build:css:ipython' 'npm run build:css:style'"`
by 
`"build:css": "concurrent \"npm run build:css:ipython\" \"npm run build:css:style\""`
Basically replace single quotes by double one and escape them. ( a real shot in the dark), 

also try to replace by
`"build:css": "npm run build:css:ipython && npm run build:css:style",`
to try to build sequentially.

And more generally if you want to try to figure it out:

```
npm install -g concurently
```
And they try various quotes see which fails
```
concurrently --raw "echo 'a'" "echo 'b'"
```

**Comment by dhirschfeld** [comment]
The single-quote inside double quotes does seems to work?
```
λ concurrently --raw "echo 'a'" "echo 'b'"
'a'
'b'
```
...trying the suggest `package.json` modifications now

**Comment by dhirschfeld** [comment]
NB: The existing `package.json` seems to mix the two usages (single-quotes / escaped double-quotes)
```
    "build:watch": "concurrent \"npm run build:css:watch\" \"npm run build:js:watch\"",
    "build": "npm run build:js && npm run build:css",
    "build:css": "concurrent 'npm run build:css:ipython' 'npm run build:css:style'",
```

**Comment by dhirschfeld** [comment]
Looks like that worked:
```
running css
> npm run build:css

> jupyter-notebook-deps@4.0.0 build:css C:\dev\code\notebook
> concurrent "npm run build:css:ipython" "npm run build:css:style"

[1]
[1] > jupyter-notebook-deps@4.0.0 build:css:style C:\dev\code\notebook
[1] > lessc --include-path=notebook/static notebook/static/style/style.less notebook/static/style/style.min.css
[1]
[0]
[0] > jupyter-notebook-deps@4.0.0 build:css:ipython C:\dev\code\notebook
[0] > lessc --include-path=notebook/static notebook/static/style/ipython.less notebook/static/style/ipython.min.css
[0]
[0] npm run build:css:ipython exited with code 0
[1] npm run build:css:style exited with code 0
```

**Comment by Carreau** [comment]
closed by jupyter/notebook#1494

**Comment by dhirschfeld** [comment]
Unfortunately when trying to run the `jupyter notebook` from source I get a blank page and the following error in the console:
![image](https://cloud.githubusercontent.com/assets/881019/15594465/fd49aa0a-23f8-11e6-96ef-96484613df2d.png)
